CMakeLists.txt

```diff
@@ -1,84 +1,99 @@
-cmake_minimum_required(VERSION 3.6)
+cmake_minimum_required(VERSION 3.27)
```

Comment:
Are you sure this change is required?

Reply:
Sort of. There was a change in how cmake deals with getting the python interpreter in version `3.27`. This is one way to enable the new behaviour, the other is to set a cmake policy. Without either of these the python executable won't be detected automatically. I'm happy to change it to using the policy method.

Reply:
No. In that case it is fine !